MabiPale2.Plugins.PacketAnalyzer/Packets/ConditionUpdate.cs

```diff
@@ -13,6 +13,11 @@ public string AnalyzePacket(PalePacket palePacket)
 			// [200300, NA262 (2017-10-20)]
 			// The condition format changed, first, with the longs, is the
 			// old one, the other is the new one.
+
+			// [(2020-05-30)]
```

Comment:
Please note that this comment style is for version changes, so they're easily recognizable and can be searched for, not for random code changes.

Regarding the actual change, I see how these values not being in Mabi.dll anymore would be a problem, but disabling this code will break parsing of older packets, which is unfortunate. Ideally we'd want to keep supporting both, which is what all those type checks are for after all. We should either readd the enums to Mabi.dll or add them to Pale.

Reply:
I just updated it to support the older versions again